lighthouse-cli/test/fixtures/perf/delayed-element.js

```diff
@@ -28,11 +28,16 @@ async function rerender(iterations) {
 setTimeout(() => {
   const imgEl = document.createElement('img');
   imgEl.src = '../dobetterweb/lighthouse-480x318.jpg';
-  const textEl = document.createElement('span');
+  const textEl = document.createElement('div');
   textEl.textContent = 'Sorry!';
+  textEl.style.height = '18px' // this height can be flaky so we set it manually
```

Comment:
L31 and L33 changes aren't part of the fix but will avoid a merge conflict we gotta deal with. 😎